MySolution.rb

```diff
@@ -0,0 +1,36 @@
+class MySolution
+
+    #problem1
+    def sumOfEven(array)
+        array.select{|v| v.even?}.sum
+    end
+    
+    #problem2
+    def getLargest2Numbers(array)
+        bigMax = -1/0.0
+        smallMax = -1/0.0
+        for i in array
+            if i > bigMax and i > smallMax 
+                smallMax = bigMax
+                bigMax = i  
+            else
+                if i > smallMax 
+                    smallMax = i 
+                end
+            end
+        end
+        result = [smallMax, bigMax]
+        return result
+    end
+
+    #problem3
+    def hasDublicates(array)
+        for i in 0..array.length 
```

Comment:
Emphasizing the confusion again for how you use `i`. This time, it's an item in an array of indices, used to access the array in the inner loop.